types/tizen-tv-webapis/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Remove all these rules and fix the errors instead.

Reply:
Is it essential?? I already had released `@types/tizen-common-web` with same rules.

Reply:
Yes fixing this is essential. New packages cannot edit this file and need to fix any reported errors.

Reply:
But in the [#PR45813](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45813), it was merged with rules like:
``` json
//tslint.json
{
    "extends": "dtslint/dt.json",
    "rules":{
        "max-line-length": false,
        "no-empty-interface": false,
        "interface-over-type-literal": false,
        "npm-naming": false,
        "no-single-declare-module": false
    }
}
```
I can't share the `widl`s to you, but there are jsdoc in each module. And I don't wanna change the doc because of committing DT. The `widl`s were written by several developers, it take a long time to fix it. So can you keep this `tslint.json`?